packages/logger/src/index.ts

```diff
@@ -0,0 +1,25 @@
+// Copyright IBM Corp. 2013,2017. All Rights Reserved.
+// Node module: @loopback/logger
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {Logger} from './keys';
+import {Component} from '../../core';
+import {PinoLoggerProvider, SetupPinoHttpLogger} from './providers/LoggerProvider';
+
+const setupLoggerKey = Logger.SequenceActions.SETUP_LOGGER;
+const loggerProviderKey = Logger.System.LOGGER_PROVIDER;
+
+export {Logger} from './keys';
+export {SimpleLogger} from './providers/LoggerProvider';
+
+export class PinoLoggerComponent implements Component {
+  providers = {
+    // PinoLoggerProvider is bound to the key [loggerProviderKey]
+    // in the client's application context.
```

Comment:
To be honest, I see very little value in this comment - it's just describing how components work.

The more interesting question is what is PinoLoggerProvider good for, where would it people use? This is not clear to me (I haven't read the tests yet), a comment explaining _that_ would be helpful.

The same comment applies to setupLoggerKey. I know what's that for because it's based on our past discussion, but people reading this code may not know.